src/GRA.Domain.Service/PerformerSchedulingService.cs

```diff
@@ -567,6 +591,12 @@ public async Task<int> GetPerformerSelectionCountAsync(int performerId)
             return await _psBranchSelectionRepository.GetCountByPerformerIdAsync(performerId);
         }
 
+        public async Task<ICollection<PsBranchSelection>> GetPerformerSelectionAsync(int performerId)
+        {
+            VerifyManagementPermission();
+            return await _psBranchSelectionRepository.GetByPerformerIdAsync(performerId);
+        }
+
```

Comment:
GetPerformerBranchSelectionsAsync performs the same function and can be used in place of this